docs/reference/landing-page.asciidoc

```diff
@@ -240,4 +241,19 @@ <h4 class="mt-3">
   </div>
 </div>
 
-<p class="my-4"><a href="https://www.elastic.co/guide/index.html">View all Elastic docs</a></p>
+<script>
+window.addEventListener("DOMContentLoaded", (event) => {
+  const left_col = document.getElementById("left_col")
+  left_col.classList.remove('col-0')
+  left_col.classList.add("col-12", "col-md-4", "col-lg-3", "h-almost-full-md", "sticky-top-md")
```

Comment:
```suggestion
  left_col.classList.add("col-12", "order-2", "col-md-4", "col-lg-3", "h-almost-full-md", "sticky-top-md")
```

Reply:
Maybe this was the intention, but on a phone, the TOC is rendered before the content (contrary to the other pages, where the TOC is rendered after the content). I personally like after better, which we can do by adding this CSS class to the left column.

Reply:
Good feedback. Thanks!

Reply:
I ended up reverting this change with https://github.com/elastic/elasticsearch/commit/52a915bf31eabefc46d1a8e2c7e2c6f8ca82c76c.

It looks like the class also changes the order on larger screens. This seems like a relatively minor detail so I don't think it's worth addressing right now.